.github/sync-repo-settings.yaml

```diff
@@ -36,10 +36,6 @@ branchProtectionRules:
       - lint
       - OwlBot Post Processor
       - 'Kokoro - Test: Integration'
-      - 'Kokoro - Test: Java 17 GraalVM Native Image - AI Platform'
-      - 'Kokoro - Test: Java 17 GraalVM Native Image - Compute'
-      - 'Kokoro - Test: Java GraalVM Native Image - AI Platform'
-      - 'Kokoro - Test: Java GraalVM Native Image - Compute'
```

Comment:
I may have made a mistake by removing the old builds prior to removing the 'required' entries here.
Unless there's an 'override', I may need to rollback and perform these updates in a different order.